pkg/api/api.go

```diff
@@ -69,20 +69,25 @@ type API struct {
 	RequireAllFF []featureflags.FeatureFlag
 	// PropertyNameOfIdentifier defines the id field if it's not called 'ID'
 	PropertyNameOfIdentifier string
+	// child indicates that this API has child. It is field within NewAPIs function
+	haveChild bool
```

Comment:
If this is **only** necessary to make the cache work, i would strongly suggest to go with the nocache solution for deletion.